crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -1057,6 +1158,17 @@ module deepbook::clob_v2 {
                         taker_commission,
                         maker_rebate
                     );
+                    vector::push_back(
```

Comment:
how big can this vector get?
can this be an issue?

Reply:
That's a good point! Since orders are stored as df in clob_v2, as per sui limits only upto a total of 1k dfs could be read in a single PTB. So, we can assume that number of order matches in a single txn would be less 1k(before df limit is hit), hence scalability of vector to store metadata is taken care off. Let me know if you see other concerns about scalability of vector. Thanks!